source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -13,25 +17,107 @@
         if (data) {
             this.fromJs(data);
         }
+        this.getAllCalificationsByKey = function (key) {
+            return _.map(this.califications(), function (calification) {
+                return calification.getCalificationByKey(key);
+            });
+        }
+        this.calificatorsHeaderTable = ko.computed(function () {
```

Comment:
Maybe it is simpler to keep `califications` only in the JSON Model and apply these mapping on parsing JSON, what do you think @joseguti84?